runtime/rasdump/javadump.cpp

```diff
@@ -3540,6 +3541,44 @@ JavaCoreDumpWriter::writeSharedClassSectionAllLayersStatsHelper(J9SharedClassJav
 	_OutputStream.writeCharacters("\n");
 }
 
+void
+JavaCoreDumpWriter::writeSharedClassSectionEachLayerStatsHelper(J9SharedClassJavacoreDataDescriptor* javacoreData)
+{
+	if (NULL == javacoreData) {
+		return;
+	}
+	if (NULL == _VirtualMachine->sharedClassConfig) {
+		return;
+	}
+	J9SharedClassCacheDescriptor *curCache = _VirtualMachine->sharedClassConfig->cacheDescriptorList;
+	if (NULL == curCache) {
+		return;
+	}
+	UDATA currentOSPageSize = javacoreData->currentOSPageSize;
+	I_8 layer = javacoreData->topLayer;
+	bool headerPrinted = false;
+	do {
+		if (currentOSPageSize != curCache->osPageSizeInHeader) {
+			if (!headerPrinted) {
+				_OutputStream.writeCharacters(
+						"NULL\n"
+						"1SCLTEXTCISL   Cache Info for a signale layer\n"
```

Comment:
Fixed the spelling. 

For the current OS page size, yes it is duplicated. I print it in each layer so that it is easier/obvious for other people to spot the difference. They do not need to look for the current OS page size in another section.